### ISSUE TITLE
Starter fails with lsdyna keyword *NODE_TRANSFORM

### ISSUE THREAD

**Issue by hdrz** [OpenRadioss/OpenRadioss#2581]
**Describe the bug**
When including *NODE_TRANSFORM in the input .k file, starter fails with exit code 3.

**To Reproduce**
Just use this keyword, as in the attached example.

**Expected behavior**
The nodes in the node-set should transform (in this case rotate).

**Desktop (please complete the following information):**
 - Version [commit id] - 81eea05192bf08c59d82bd11f9de24f37924e8fc
- Hardware [which CPU] - Any x86_64, tried on few, intel and amd
- Configuration [number of OpenMP threads, and MPI tasks] - Starter with any np or nt

**Additional context**
[test.zip](https://github.com/OpenRadioss/OpenRadioss/files/15215387/test.zip)

**Comment by sebastienVilleneuve** [comment]
Hi,

Rotation angle is null in your model.
only 2 first fields of the DEFINE_TRANSFORMATION card 'a1' & 'a2' are filled , then this card is understood as transformation using points 1 & 2 ( that are not existing in the model)
Could you retry with a non null value ?

regards,
Sébastien

**Comment by hdrz** [comment]
Hi,
From the user guide for the a1 field:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/29428222/d1409abf-365f-4e5e-a6d6-015666cff034)
So with only a1,a2,a3 filled, a1 and a2 hold **node** numbers, while a3 holds the angle. In the model attached nodes 1 & 2 exist.

I didn't try the full option of rotate yet, it might work, but this is definitely a bug (in the implementation or the ref guide..)

Thank you,
Hadar

**Comment by sebastienVilleneuve** [comment]
Ok , Thanks for the details,
I will check

Sébastien

**Comment by sebastienVilleneuve** [comment]
Hi,

some more information :

In Dyna manual , DEFINE_TRANSFORMATION makes reference to points, defined in option itself, not to nodes
![image](https://github.com/OpenRadioss/OpenRadioss/assets/100404030/69ccb016-ea61-4642-82ec-a8e5b7563eea)

**Comment by hdrz** [comment]
Yes that is true, but according to the ref manual of radioss it works differently in radioss. Also, the option to use POINT is not present in radioss, again according to ref manual.

Can you clarify this issue?
Thank you

**Comment by sebastienVilleneuve** [comment]
I confirm , doc is wrong ,

POINT is missing , & ROTATE is using POINTs , not nodes.
We will update the doc.

Sébastien

**Comment by hdrz** [comment]
OK that clarifies the issue.
At this point I would like to ask to add the option ROTATE3NA, which sometimes makes life a lot easier.
Thanks again!

**Comment by sebastienVilleneuve** [comment]
ROTATE3NA is already supported , this will also be updated in the doc.

Sébastien

**Comment by hdrz** [comment]
Just now tried with the option ROTATE3NA, starter stops with exit code 3.
I simply modified the test.k file and tried to run it.

However if I add two POINT lines ROTATE works beautifully.
